src/components/Checkbox/index.jsx

```diff
@@ -1,7 +1,27 @@
 import React from "react";
+import styles from "./styles/styles.module.scss";
 
-const Checkbox = () => {
-  return <p>checkbox</p>;
+const Checkbox = ({ question, options, onChange }) => {
```

Comment:
I guess it would make more sense to add `required` prop. Then you can implement the `isValid` functionality here and send it upwards via `onChange`.

Reply:
hope I got it and changed it well